.travis.yml

```diff
@@ -26,8 +26,9 @@ install:
 - npm install
 before_script:
 # Install Firefox Nightly to prevent signing errors.
+- npm i -g get-firefox
```

Comment:
if you call it from an npm script then you won't have to worry about adjusting the PATH